gradio/examples.py

```diff
@@ -55,6 +82,11 @@ def __init__(
             cache_examples: if True, caches examples for fast runtime. If True, then `fn` and `outputs` need to be provided
             examples_per_page: how many examples to show per page (this parameter currently has no effect)
         """
+        if _initiated_directly:
+            raise warnings.warn(
```

Comment:
I think we should raise an exception. The demo would be broken if they used `gr.examples.Examples` right?

Reply:
Not necessarily. They can use `ex = gr.examples.Example` as long as they remember to run `await ex.create()` afterwards. Basically just what the top-level function does.

Reply:
Actually, I just realized `raise` will actually try to raise an exception. I think you mean `warnings.warn(...)` only.

Yea that's a good point! Let's just warn instead of raising an exception.

Reply:
Ah good catch, will fix (sorry missed this earlier)